projects/storefrontlib/src/lib/ui/pages/store-finder-page/store-finder-page.module.ts

```diff
@@ -5,13 +5,50 @@ import { Routes, RouterModule } from '@angular/router';
 import { CmsPageGuards } from '../../../cms/guards';
 import { StoreFinderPageComponent } from './store-finder-page.component';
 import { StoreFinderPageLayoutModule } from '../../layout/store-finder-page-layout/store-finder-page-layout.module';
+import { StoreFinderListComponent } from '../../../store-finder/components/store-finder-list/store-finder-list.component';
+import { StoreFinderListCountComponent } from '../../../store-finder/components/store-finder-list-count/store-finder-list-count.component';
+import { StoreFinderGridComponent } from '../../../store-finder/components/store-finder-grid/store-finder-grid.component';
+// tslint:disable-next-line:max-line-length
+import { StoreFinderStoreDescriptionComponent } from '../../../store-finder/components/store-finder-store-description/store-finder-store-description.component';
 
 const routes: Routes = [
   {
     path: 'store-finder',
     canActivate: [CmsPageGuards],
     data: { pageLabel: 'storefinderPage' },
-    component: StoreFinderPageComponent
+    component: StoreFinderPageComponent,
+    children: [
+      {
+        path: 'find-stores',
+        canActivate: [CmsPageGuards],
+        data: { pageLabel: 'storefinderPage' },
+        component: StoreFinderListComponent
+      },
+      {
+        path: 'view-all-stores',
+        canActivate: [CmsPageGuards],
+        data: { pageLabel: 'storefinderPage' },
+        component: StoreFinderListCountComponent
```

Comment:
Another idea I had is maybe renaming some of the components to describe there actual use. For instance, I get mixed up between, _list_, _display-list_ and _list-count_.